frontend/src/pages/TracesExplorer/Filter/Filter.tsx

```diff
@@ -184,6 +184,13 @@ export function Filter(props: FilterProps): JSX.Element {
 					...currentQuery.builder,
 					queryData: currentQuery.builder.queryData.map((item) => ({
 						...item,
+						// on creating alert from Traces page and coming back to Traces, the count aggregateOperator causes an error, therefore, changing it back to noop
```

Comment:
@ahmadshaheer please help me understand why this hack is needed specifically in Traces Explorer and not in Logs Explorer.

Reply:
@srikanthccv , on coming back to traces page after creating alert from it, The noop would get replaced with count which would cause an issue. 

I was facing this issue only on traces page (please refer to the loom videos). therefore, added this code.

I later discussed with @YounixM , and he suggested that we have to move the modification logic inside the `useGetCompositeQueryParam`. I tried implementing it there, but I wasn't able to get the `panel type` (we wanted the query modification to happen only for list view and traces). 

I will repick fixing this issue today/tomorrow and properly fix it. 

Video 1: [Traces -> create alert -> go back -> error](https://www.loom.com/share/1ad14534aab84410a519fadd34407edd)
Video 2: [Logs -> create alert -> go back -> works fine](https://www.loom.com/share/d2a88d63ca8f4f8caf1b1896abef02c8)